src/common/extension-telemetry-events.ts

```diff
@@ -80,6 +80,7 @@ export const LEFT_NAV_PANEL_EXPANDED: string = 'leftNavPanelExpanded';
 export const NEEDS_REVIEW_TOGGLE: string = 'NeedsReviewToggled';
 export const NAVIGATE_TO_NEW_CARDS_VIEW: string = 'NavigateToNewCardsView';
 export const SET_AUTO_DETECTED_FAILURES_DIALOG_STATE: string = 'setAutoDetectedFailuresDialogState';
+export const UNHANDLED_EXCEPTION: string = 'unhandledException';
```

Comment:
I think it would be more accurate for this to read `ERROR` than `EXCEPTION` (same for the type name on L287)